### PR TITLE
Added solution

### PR DESCRIPTION
**Pull request by RobertAKARobin**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -66,7 +65,7 @@ One of the lines of code in the following snippet will throw an error. Which one
 /*8*/ }
 ```
 
-> Your answer...
+> Line 3 will throw an error. `use strict` prevents you from using any variables without declaring them first with `var`. `i` hasn't been declared.
```
Comment (amaseda) [inline readme.md]: My thoughts are that we should replace this question with a a larger topic. `"use strict";` has only come up in Angular and even then it's not essential.

Maybe a Rails question? At the moment there's only one Ruby question.

File: readme.md [hunk 2]
```diff
@@ -102,7 +119,7 @@ What is the difference between the two following lines of code?
 @artist.save!
 ```
 
-> Your answer...
+> If validations fail, the first line will fail silently, while the second line will throw a fatal error.
```
Comment (amaseda) [inline readme.md:143]: Disregard part of my earlier comment - here's a Rails-y question!

I think we'd learn more about where the students are at if we quiz them on ActiveRecord on a more fundamental level. How about asking them to list the Rails equivalents of some Mongoose methods (and/or vice-versa)?

File: readme.md [hunk 3]
```diff
@@ -124,14 +171,46 @@ Instantiate an instructor named 'Andy' and call its `receivePresent` method with
 ```js
 // Your code starts here...
 
-```
+function Instructor(name){
+  this.name = name;
+}
+Instructor.prototype.receivePresent = function(gift){
+  console.log(this.name + " promptly drops the " + gift + " on the floor.");
+}
```
Comment (nolds9) [inline readme.md:179]: pure comedic, gold

File: readme.md (hunk 2 above)
Comment (andrewsunglaekim) [inline readme.md:143]: :+1: ^

File: readme.md [hunk 4]
```diff
@@ -10,6 +10,30 @@
...
+  <head>
+    <title>Quiz
+  </head>
+  <body>
+    <h1>What a great site.</h1>
+  </body>
+</html>
+```
+
+> The `<title>` element has no closing tag.
```
Comment (jshawl) [inline readme.md:27]: this made me feel stupid

**Pull request closed without merging**